source/install/prepare-mattermost-database.rst

```diff
@@ -124,16 +188,16 @@ To set up a PostgreSQL database for use by the Mattermost server:
 
       .. code-block:: none
         :class: mm-code-block 
-    
+
           psql --dbname=mattermost --username=mmuser --password
 
   .. tab:: Multi-server deployment
```

Comment:
For me this not a cause of multi-server deployment, rather better phrased as "db on the same server as mattermost" or "db on a remote system".

0/5 on the terms but the concept is there.

Reply:
@coltoneshaw - What terminology do you tend to hear from customers when differentiating between a deployment where all components are on a single server versus components spread across servers?

Reply:
Oh, I'd +1 neil on this one. Although, both are technically correct. I'd lean towards `DB on another server`